### PR TITLE
Fix for MissingPropertyException during startup

### PR DESCRIPTION
**Pull request by fernau**
This fixes an MissingPropertyException reported with some versions of artifactory.

### CODE REVIEW

**Review by DanielRuaMadrid — APPROVED** [review]
LGTM

File: plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy [hunk 1]
```diff
@@ -148,14 +147,24 @@ private runHandlerWithInitialisation(Map params) {
   }
 }
 
+private String getArtifactoryVersion() {
+  try {
+    return ctx.versionProvider?.running?.versionName
```
Comment (vladt) [inline plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy:152]: So we'll always get "unknown" for newer version?

Comment (fernau) [reply plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy:152]: Yes, for now this only allows the plugin to start if there is an API change.
We still need to test out what has changed in which version.

Comment (fernau) [reply plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy:152]: I full fix might be more work. created new ticket: https://issues.sonatype.org/browse/CLM-22327

**Review by vladt — APPROVED** [review]
+1

**Pull request merged**